app/src/main/java/com/kelvin/bdaypost/birthday/data/model/Month.kt

```diff
@@ -0,0 +1,29 @@
+package com.kelvin.bdaypost.birthday.data.model
+
+object Month {
```

Comment:
Can month be an enum class with a name and number of days instead? Something like
```
enum class Month(name: String, numDays: Int) {
    JAN("JAN", 31)
}
```